.changelog/2459.feature.md

```diff
@@ -0,0 +1,2 @@
+When registering a new runtime, check whether the given key manager ID points
```

Comment:
This is actually `breaking` as the same transaction can be processed differently by different versions.